### PR TITLE
Manipulation: Allow to restore the legacy jQuery.htmlPrefilter

### PR DESCRIPTION
**Pull request by mgol**
This goes in tandem with https://github.com/jquery/jquery.com/pull/196.

### CODE REVIEW

**Comment by mgol** [comment]
This goes in tandem with https://github.com/jquery/jquery.com/pull/196.

File: src/manipulation.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\/\0>\x20\t\r\n\f]*)[^>]*)\/>/gi;
+
+jQuery.restoreLegacyInsecureHtmlPrefilter = function() {
```
Comment (mgol) [inline src/manipulation.js]: Optionally, to make people really think before invoking that, we can do:
```suggestion
jQuery.UNSAFE_restoreLegacyInsecureHtmlPrefilter = function() {
```

Comment (dmethvin) [reply src/manipulation.js]: Maybe `UNSAFE_restoreHtmlPrefilter`?

Comment (mgol) [reply src/manipulation.js]: @dmethvin That seems to suggest to me that we don't have `htmlPrefilter` now but we do, just with a different value.

Comment (dmethvin) [reply src/manipulation.js]: How about just `jQuery.UNSAFE_restoreLegacyHtmlPrefilter` since we have UNSAFE in the name now.

Comment (mgol) [reply src/manipulation.js]: PR updated.

**Comment by mgol** [comment]
@dmethvin What renaming? I've renamed it a few hours before your PR approval...

**Comment by dmethvin** [comment]
Strange, I must have commented on the earlier commit. In any case, this looks ready to go.

**Pull request merged**